src/main/java/core/basesyntax/strategy/operation/BalanceOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+
+public class BalanceOperation implements OperationHandler {
+    private StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public void operationType(String fruit, int amount) {
```

Comment:
The method name `operationType` is not descriptive of what the method actually does. Consider renaming it to something more indicative of its function, like `applyBalance`.